hack/update-staging-godeps-dockerized.sh

```diff
@@ -75,8 +75,26 @@ function updateGodepManifest() {
     rm -rf Godeps # remove the current Godeps.json so we always rebuild it
     GOPATH="${TMP_GOPATH}:${GOPATH}:${GOPATH}/src/k8s.io/kubernetes/staging" godep save ${GODEP_OPTS} ./... 2>&1 | sed 's/^/  /'
 
-    # Rewriting Godeps.json to remove commits that don't really exist because we haven't pushed the prereqs yet
-    go run "${KUBE_ROOT}/staging/godeps-json-updater.go" --godeps-file="${TMP_GOPATH}/src/k8s.io/${repo}/Godeps/Godeps.json" --override-import-path="k8s.io/${repo}"
+    # Rewriting Godeps.json to cross-out commits that don't really exist because we haven't pushed the prereqs yet
+    local repo
+    for repo in $(ls -1 ${KUBE_ROOT}/staging/src/k8s.io); do
+      # remove staging prefix
+      jq '.Deps |= map(.ImportPath |= ltrimstr("k8s.io/kubernetes/staging/src/"))' Godeps/Godeps.json |
```

Comment:
Is this a new dependency?

Reply:
no, it's inside docker and we use jq elsewhere inside the container.